container/Compiler.cs

```diff
@@ -860,6 +865,18 @@ private bool IsBoolean(SyntaxNode node)
                 return type?.SpecialType is SpecialType.System_Boolean;
             }
 
+            private bool HasReturnValue(ExpressionSyntax node)
+            {
+                var typeInfo = _semanticModel.GetTypeInfo(node);
+                var type = typeInfo.Type;
+                // Error types (undefined functions like println) are treated as void
+                if (type == null || type.TypeKind == TypeKind.Error)
+                {
+                    return false;
+                }
```

Comment:
`println`, `print`, `WriteLine` symbols are emits via preamble tree from `CSharpCompilation.Create` in future.
Currently this implements are acceptable.